service/src/main/java/org/apache/griffin/core/job/JobController.java

```diff
@@ -113,4 +113,10 @@ public JobHealth getHealthInfo() {
                 .contentType(MediaType.APPLICATION_OCTET_STREAM)
                 .body(resource);
     }
+
```

Comment:
We have to add this method to our documentation and create new postman examples.

Reply:
I updated api guide and postman collection